sympy/functions/elementary/complexes.py

```diff
@@ -838,6 +992,12 @@ class polar_lift(Function):
     >>> polar_lift(4*p)
     4*p
 
+    Parameters
+    ==========
+
+    arg : Expr
+        Real, complex or mixed expression.
```

Comment:
I don't think that `mixed` is a clear notation because real numbers and real numbers combined with imaginary numbers all belongs to complex numbers. You should better drop that and denote like `Real or complex expression.`

Reply:
yes.